fabfile/add.py

```diff
@@ -0,0 +1,92 @@
+import os.path
+
+from fabric.api import task, cd, path, run, runs_once, roles, sudo
+
+import utils
+import config
+
+@task
+def session_analytics(*args):
+    'Adds the session_analytics extension to the instance in pg-latest'
+    # TODO: Requires hstore, do we install it automatically?
+    utils.add_github_to_known_hosts() # make sure ssh doesn't prompt
+    repo = config.paths['session-repo']
+    prefix = config.paths['pg-latest']
+
+    if len(args) == 0:
+        git_ref = 'master'
+    else:
+        git_ref = args[0]
+
+    run('rm -rf {} || true'.format(repo)) # -f because git write-protects files
+    run('git clone -q git@github.com:citusdata/session_analytics.git {}'.format(repo))
+    with cd(repo), path('{}/bin'.format(prefix)):
+        run('git checkout {}'.format(git_ref))
+        run('make install')
+
+    # TODO: What if the server isn't running?
+    with cd(prefix):
+        run('bin/psql -c "CREATE EXTENSION session_analytics;"')
+
+@task
+def hll():
+    'Adds the hll extension to the instance in pg-latest'
```

Comment:
Minor: What about moving function comments to just before the function definition as we do on Citus repo?